extensions/datasource/runtime/src/main/java/io/quarkus/datasource/runtime/DataSourceBuildTimeConfig.java

```diff
@@ -16,4 +16,12 @@
     @ConvertWith(DatabaseKindConverter.class)
     public Optional<String> dbKind = Optional.empty();
 
+    /**
+     * Whether this particular data source should be excluded from the health check if
+     * the general health check for data sources is enabled. By default, the health check includes
+     * all configured data sources (if it is enabled).
+     */
```

Comment:
```suggestion
    /**
     * Whether this particular data source should be excluded from the health check if
     * the general health check for data sources is enabled. 
     * <p>
     * By default, the health check includes all configured data sources (if it is enabled).
     */
```

Introduce `<p>` to render out nicely the generated doc.